recipes/namd/3.0b6/gh200/modules.yaml

```diff
@@ -0,0 +1,23 @@
+modules:
+  prefix_inspections:
+    bin:
+      - PATH
+    lib:
+      - LD_LIBRARY_PATH
+    lib64:
+      - LD_LIBRARY_PATH
+
+  default:
+    arch_folder: false
+    roots:
+      tcl: /user-environment/modules
+    tcl:
+      all:
+        autoload: none
+      hash_length: 0
+      exclude_implicits: true
+      exclude: ['%gcc@7.5.0', 'gcc %gcc@7.5.0']
+      projections:
+        all: '{name}/{version}'
+        #charmpp backend=multicore: '{name}/multicore/{version}'
+        #namd ^charmpp backend=multicore: '{name}/multicore/{version}'
```

Comment:
are these needed?

Reply:
Not for this particular environment, but they would be useful when we will add a multi-node one. I can remove them for the time being.